docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -145,115 +166,116 @@ can set the following in your ``galaxy.yml`` file:
+Install the tarball into a directory configured in :ref:`COLLECTIONS_PATHS` so Ansible can easily find and load the collection. If you do not specify a path value, ``ansible-galaxy collection install`` installs the collection in the first path defined in :ref:`COLLECTIONS_PATHS`.
+
+.. _collections_scm_install:
+
+Installing your collection locally from a git repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To install your collection locally from a git repository, specify the repository and the branch you want to install:
 
 .. code-block:: bash
 
-   ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections
+   collection_dir#> ansible-galaxy collection install git+https://github.com/org/repo.git,devel
 
+.. include:: ../shared_snippets/installing_collections_git_repo.txt
 
-You should use one of the values configured in :ref:`COLLECTIONS_PATHS` for your path. This is also where Ansible itself will
-expect to find collections when attempting to use them. If you don't specify a path value, ``ansible-galaxy collection install``
-installs the collection in the first path defined in :ref:`COLLECTIONS_PATHS`, which by default is ``~/.ansible/collections``.
+Reviewing your collection
+-------------------------
 
-If you want to use a collection directly out of a checked out git repository, see :ref:`hacking_collections`.
+Next, review the collection:
 
-Next, try using the local collection inside a playbook. For examples and more details see :ref:`Using collections <using_collections>`
+* Run a playbook that uses the modules and plugins in your collection. Verify that new features and functionality work as expected. For examples and more details see :ref:`Using collections <using_collections>`. 
+* Check the documentation for typos.
+* Check that the version number of your tarball is higher than the latest published version on the distribution server or servers.
+* If you find any issues, fix them and rebuild the collection tarball.
 
-.. _collections_scm_install:
+.. _collection_versions:
 
-Trying your collection from a git repository
---------------------------------------------
+Understanding collection versioning
+-----------------------------------
 
-You can also test a version of your collection in development by installing it from a git repository.
+The only way to change a collection is to release a new version. The latest version of a collection (by highest version number) is the version displayed everywhere in Galaxy and Automation Hub. Users can still download older versions.
 
-.. code-block:: bash
+Follow semantic versioning when setting the version for your collection. In summary:
 
-   ansible-galaxy collection install git+https://github.com/org/repo.git,devel
+* Increment major (for example: x in `x.y.z`) version number for an incompatible API change.
```

Comment:
I think it would be best to avoid for example: in the parentheses. If there is an alternate way, I'd suggest using it to avoid the use of parentheses in this manner.

Reply:
Fixed in 2b84951fc61422551f159ccf5df686beff5fb23e.